Xamarin.Forms.sln

```diff
@@ -1359,7 +1359,6 @@ Global
 		{411B960D-6D30-4079-83B2-ABB9987D2EDF}.Release|x86.ActiveCfg = Release|Any CPU
 		{411B960D-6D30-4079-83B2-ABB9987D2EDF}.Release|x86.Build.0 = Release|Any CPU
 		{AFF3AD0D-7181-4551-A29C-8701FE3E6753}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{AFF3AD0D-7181-4551-A29C-8701FE3E6753}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
that, and the other changes in the sln, is disabling Pages and ControlGallery for Tizen

Reply:
@StephaneDelcroix Pages and ControlGallery projects for Tizen can be built in my environment without any errors. Could you share if you have any error messges or log file?

Reply:
@WonyoungChoi from within VS ?

Reply:
@StephaneDelcroix Yes I tried in VS2017 15.8.5.